src/one_dragon/base/controller/pc_game_window.py

```
@@ -125,32 +127,69 @@ def active(self) -> bool:
+
+    def _minimize_other_windows(self) -> None:
+        """通过任务栏命令最小化其他窗口。"""
         try:
-            win.restore()
-            win.activate()
-            return True
+            shell_hwnd = win32gui.FindWindow('Shell_TrayWnd', None)
+            if shell_hwnd:
+                win32gui.SendMessage(shell_hwnd, win32con.WM_COMMAND, 419, 0)
+                time.sleep(0.5)
+        except Exception:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)